services/datamanager/datasync/upload_arbitrary_file.go

```diff
@@ -22,7 +22,7 @@ func uploadArbitraryFile(ctx context.Context, client v1.DataSyncServiceClient, f
 	md := &v1.UploadMetadata{
 		PartId:        partID,
 		Type:          v1.DataType_DATA_TYPE_FILE,
-		FileName:      filepath.Base(f.Name()),
+		FileName:      f.Name(),
```

Comment:
One question based on the doc string: does this mean this will always return the value that we passed into os.Open? So if we ever opened this with a relative path for whatever reason (say down the line, we updated this to use the relative path), would this then return the relative path?

Reply:
Good point! Grab `filepath.Abs` now and tested with absolute paths (e.g. `/tmp/my/path`) and relative paths (e.g. `mypath`) which correctly save to `/tmp/my/path` and `/Users/alexagreenberg/viam/rdk/mypath`